kasa/tests/fixtures/HS300(US)_2.0_1.0.12.json

```diff
@@ -0,0 +1,90 @@
+{
+    "emeter": {
+        "get_realtime": {
+            "current_ma": 6,
+            "err_code": 0,
+            "power_mw": 277,
+            "slot_id": 0,
+            "total_wh": 62,
+            "voltage_mv": 120110
+        }
+    },
+    "system": {
+        "get_sysinfo": {
+            "alias": "#MASKED_NAME#",
+            "child_num": 6,
+            "children": [
+                {
+                    "alias": "#MASKED_NAME#",
+                    "id": "8006A0F1D01120C3F93794F7AACACDBE1EAD246D00",
+                    "next_action": {
+                        "type": -1
+                    },
+                    "on_time": 710216,
+                    "state": 1
+                },
+                {
+                    "alias": "#MASKED_NAME#",
+                    "id": "8006A0F1D01120C3F93794F7AACACDBE1EAD246D01",
+                    "next_action": {
+                        "type": -1
+                    },
+                    "on_time": 710216,
+                    "state": 1
+                },
+                {
+                    "alias": "#MASKED_NAME#",
+                    "id": "8006A0F1D01120C3F93794F7AACACDBE1EAD246D02",
+                    "next_action": {
+                        "type": -1
+                    },
+                    "on_time": 710216,
+                    "state": 1
+                },
+                {
+                    "alias": "#MASKED_NAME#",
+                    "id": "8006A0F1D01120C3F93794F7AACACDBE1EAD246D03",
+                    "next_action": {
+                        "type": -1
+                    },
+                    "on_time": 710216,
+                    "state": 1
+                },
+                {
+                    "alias": "#MASKED_NAME#",
+                    "id": "8006A0F1D01120C3F93794F7AACACDBE1EAD246D04",
+                    "next_action": {
+                        "type": -1
+                    },
+                    "on_time": 710216,
+                    "state": 1
+                },
+                {
+                    "alias": "#MASKED_NAME#",
+                    "id": "8006A0F1D01120C3F93794F7AACACDBE1EAD246D05",
```

Comment:
I'm wondering if we should scrub these, too, assuming it's prefixed with the device id or something.

Reply:
I'm honestly not sure what is its, but it might be nice to keep it to tell them apart in testing

Reply:
Okay, let's keep them for now!